packages/service/src/storybook.utils.ts

```diff
@@ -0,0 +1,380 @@
+import type { Logger } from '@wdio/logger'
```

Comment:
How about move this into `/packages/service/src/storybook/utils.ts` and have the types live next to it `/packages/service/src/storybook/types.ts`

Reply:
@christian-bromann, do you mean to move all files into a storybook folder?

Reply:
yeah, instead of having two files with storybook in their name, have a storybook folder with files with shorter file names in there

Reply:
thank, will do that in a few minutes

Reply:
Pushed the changes, thanks